spec/rubocop/cop/lint/useless_access_modifier_spec.rb

```diff
@@ -291,6 +291,28 @@
     end
   end
 
+  context 'when using ActiveSupport behavior when Rails is not eabled' do
+    it 'reports offenses' do
+      src = ['module SomeModule',
+             '  extend ActiveSupport::Concern',
+             '  class_methods do',
+             '    def some_public_class_method',
+             '    end',
+             '    private',
+             '    def some_private_class_method',
+             '    end',
+             '  end',
+             '  def some_public_instance_method',
+             '  end',
+             '  private',
+             '  def some_private_instance_method',
+             '  end',
+             'end']
+      inspect_source(cop, src)
+      expect(cop.offenses.size).to eq(1)
```

Comment:
Sorry, @maxjacobson you're totally right, I wasn't quite awake this morning and I misread the code!